samples/apps/smallbank/smallbank.cmake

```diff
@@ -49,38 +61,26 @@ if(BUILD_TESTS)
         --metrics-file small_bank_${CONSENSUS}_metrics.json
     )
 
-  endforeach()
-
-  if(${SERVICE_IDENTITY_CURVE_CHOICE} STREQUAL "secp256k1_bitcoin")
-    set(SMALL_BANK_SIGNED_VERIFICATION_FILE
-        ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank_50k.json
-    )
-    set(SMALL_BANK_SIGNED_ITERATIONS 50000)
-  else()
-    set(SMALL_BANK_SIGNED_VERIFICATION_FILE
-        ${CMAKE_CURRENT_LIST_DIR}/tests/verify_small_bank_2k.json
+    # These tests require client-signed signatures: - HTTP C++ perf clients
```

Comment:
I know we've been discussing this before but I believe that this now works with PBFT?

Reply:
I think the request signatures are only verified on the forwardee node though.

Reply:
PBFT skips forwarding though so I don't think there is a point in running this test

Reply:
PBFT uses its own forwarding path. However, this is the only performance we have with client signatures so I think we should enable it?

Reply:
Sorry I got confused between this test and `small_bank_sigs_forwarding`. This comment should go though.

Reply:
We should still run a test with client signatures against PBFT, it's useful to know what the performance of that is, whether the signatures can be offloaded or not.

Reply:
Yep. This test is enabled with PBFT now though. 
![image](https://user-images.githubusercontent.com/42961061/76750302-ed66e700-6775-11ea-94fa-71a10724c14d.png)

Reply:
Oh right, sorry about that then.